ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,48 @@
+## Description
+
+Provide a description of your issue.
+For any suggestion, please address them on the [forum](https://forum.freegamedev.net/viewforum.php?f=16).
+For translations, go to [transifex](https://www.transifex.com/supertuxkart/supertuxkart/).
+
+## Steps to reproduce
+
+## Configuration
+
+STK release version :
```

Comment:
Generally the : comes right afterwards without a space, e.g. STK release version: